scipy/optimize/_optimize.py

```diff
@@ -2928,9 +2941,17 @@ def bracket(func, xa=0.0, xb=1.0, args=(), grow_limit=110.0, maxiter=1000):
     >>> plt.plot(xc, fc, "bx")
     >>> plt.show()
 
+    Note that both initial points were to the right of the minimum, and the
+    third point was found in the "downhill" direction: the direction
+    in which the function appeared to be decreasing (to the left).
+    The final points are strictly ordered, and the function value
+    at the middle point is less than the function values at the endpoints;
+    it follows that a minimum must lie within the bracket.
+
     """
     _gold = 1.618034  # golden ratio: (1.0+sqrt(5.0))/2.0
     _verysmall_num = 1e-21
+    xa, xb = np.asarray([xa, xb])
```

Comment:
Do we need this line as `xa` and `xb` are advertised as being floats?

Reply:
I can guess why I did it, but if I don't remember and I didn't test it, it must not have been important. I'll remove it.

Reply:
Ah ok, neat trick